packages/api-gen/README.md

```diff
@@ -35,6 +41,21 @@ options:
 
 ```bash
 pnpx @shopware/api-gen loadSchema --help
+
+# load schema from store API
+pnpx @shopware/api-gen loadSchema --apiType=store --filename=storeApiSchema.json
+
+# load schema from admin API
+pnpx @shopware/api-gen loadSchema --apiType=admin --filename=adminApiSchema.json
+```
+
+Remember to add `.env` file in order to authenticate with Shopware instance.
+
+```bash
+OPENAPI_JSON_URL="https://demo-frontends.shopware.store"
+OPENAPI_ACCESS_KEY="SWSCBHFSNTVMAWNZDNFKSHLAYW"
```

Comment:
We should use placeholders instead of real data, even if they are just test data. It can lead to unnecessary questions